static/scripts/experience.js

```diff
@@ -1,23 +1,86 @@
-const $ = window.$;
+//const $ = window.$;
 
+let stagedExp = [];
+let stagedObj = [];
+let userId;
 $(document).ready(function () {
-    $('.tabs .tab').click(function(){
-	if ($(this).hasClass('signin')) {
-            $('.tabs .tab').removeClass('active');
-            $(this).addClass('active');
-            $('.cont').hide();
-            $('.signin-cont').show();
-	}
-	if ($(this).hasClass('signup')) {
-            $('.tabs .tab').removeClass('active');
-            $(this).addClass('active');
-            $('.cont').hide();
-            $('.signup-cont').show();
-	}
+	userId = JSON.parse(atob(localStorage.getItem('data')))['user_id'];
+	//localStorage.removeItem('data');
+    /* Add positive button clicked
+     * Positive Experience Button Clicked adds to build stagedObj*/
+    $("#positive-add").on('click', function () {
+		const name = $('#positive').val()
+		let scale = $('#positive_scale').val()
+
+		queueExp(name, scale, 'positive');
+		$('#positive').val('');
+		$('#positive_scale').val('');
+	});
+
+	$('#stow_button').on('click', function () {
+		save_experiences(stagedObj);
+		$("#experienceAdded").empty();
+		stagedExp = []
+		stagedObj = []
+	});
+
+    /* Add negative button clicked
+     * Negative Experience Button Clicked adds to build stagedObj*/
+    $("#negative-add").click(function(){
+        let name = $('#negative').val();
+        let scale = $('#negative_scale').val();;
+		queueExp(name, scale, 'negative')
+		$('#negative').val('');
+		$('#negative_scale').val('');
     });
-    $('.container .bg').mousemove(function(e){
-	var amountMovedX = (e.pageX * -1 / 30);
-	var amountMovedY = (e.pageY * -1 / 9);
-	$(this).css('background-position', amountMovedX + 'px ' + amountMovedY + 'px');
+
+    /* 'Remove' icon clicked
+     * Deletes stagedObj that was clicked*/
+    $('body').on('click', '.remove-icon', function() {
+	/*Delete the element*/
+	$(this).parent().fadeOut(300);
+	let index = stagedExp.indexOf($(this).children().attr("exp_name"));
+	stagedExp.splice(index, 1);
+	/*update the stagedObj*/
+	for (let i = 0; i < stagedObj.length; i++) {
+		console.log(stagedObj[i].name); debugger;
```

Comment:
@dbconfession78 remove debugger tag
and console.log()

Reply:
Done